include/SFML/Graphics/Transform.inl

```diff
@@ -55,7 +55,7 @@ constexpr Transform::Transform(float a00, float a01, float a02,
 ////////////////////////////////////////////////////////////
 constexpr const float* Transform::getMatrix() const
```

Comment:
I think it's fine as-is. If we change to something like a `std::array` return type then we're returning an entire copy of the array instead of just a view into that array which is unnecessary work.

Reply:
Ideally we make a note to change this to a `std::span<float, 16>` in SFML 4 once we can get our hands on C++20 features.